tools/Makefile.am

```diff
@@ -139,7 +139,8 @@ crm_rule_SOURCES 	= crm_rule.c
 crm_rule_LDADD		= $(top_builddir)/lib/cib/libcib.la		\
 			  $(top_builddir)/lib/pengine/libpe_rules.la   \
 			  $(top_builddir)/lib/pengine/libpe_status.la   \
-			  $(top_builddir)/lib/common/libcrmcommon.la
+			  $(top_builddir)/lib/common/libcrmcommon.la	\
+			  $(top_builddir)/lib/pacemaker/libpacemaker.la
```

Comment:
libraries are listed highest-to-lowest level left-to-right (really only has an impact when linking statically, but it's good habit)

Reply:
So libpacemaker first as the highest-level library? Not a pattern that's consistently followed so I wasn't aware of it. Easy switch.